client/src/lib/components/sandbox/play/Arena.svelte

```diff
@@ -49,11 +93,36 @@
 		} else {
 			hideHoveredPieceTooltip();
 		}
+		if (issuingOrder) {
+			switch(currentPhaseName) {
```

Comment:
Not sure if it's necessary here, but just a reminder that you can always break things into further components - e.g. the `Arena` component could actually have one file per phase, if it reaches a point where you are worried about defining too many functions or branches.